### ISSUE TITLE
KL loss Query

### ISSUE THREAD

**Issue by Yumlembam** [Flawless1202/VGAE_pyG#1]
Can you please reply what is kl loss  kl_loss = 1 / x.size(0) * self.kl_loss() in the loss function. It will be very helpful.

**Comment by chansigit** [comment]
the same question.

I think the kl-loss shouldn't be divided by x.size(0) since VGAE's implementation used row means.

**Comment by TommyGiak** [comment]
It is the Kullback–Leibler divergence between the z distribution and a standard Normal distribution.
For more details look at this:
https://towardsdatascience.com/understanding-variational-autoencoders-vaes-f70510919f73
